cmake/storage_dependencies.cmake

```diff
@@ -66,6 +66,7 @@ endforeach()
 
 
 ################### gRPC ####################
+set(MODYN_GRPC_CPP_PLUGIN "grpc_cpp_plugin")  # default, without gRPC:: prefix
```

Comment:
I'll remove this, auto merge didn't remove this properly.

Reply:
These are conflicts with the integrationtest PR you merged? I am a little confused, I remember seeing these gRPC changes before

Reply:
I have merged #432  into this PR before Removing those things and merging #432 to main. I guess git just got confused when I merged main into this PR after #432 was merged into main. I didn't want to touch those files in this PR at all.